examples/grpc/grpc_example.cfg

```
@@ -0,0 +1,21 @@
+
+grpc_port: 9314
+
+probe {
+  name: "grpc_generic"
+  type: GRPC
+  targets {
+    # Hit another cloudprober instance running with grpc_server enabled.
+    host_names: "localhost:9314"
+  }
+
+  grpc_probe {
+    method: GENERIC
+      request {
+        insecure_transport: true
+        
+        call_service_method: "cloudprober.Cloudprober.ListProbes"
+        body: "{}"
+      }
```

Sorry didn't notice this earlier, but insecure_transport is a `grpc_probe` level field.
```suggestion
    insecure_transport: true
    method: GENERIC
    request {
      call_service_method: "cloudprober.Cloudprober.ListProbes"
      body: "{}"
    }
```